life_expectancy/tests/conftest.py

```diff
@@ -21,3 +21,14 @@ def run_before_and_after_tests() -> None:
 def pt_life_expectancy_expected() -> pd.DataFrame:
     """Fixture to load the expected output of the cleaning script"""
     return pd.read_csv(FIXTURES_DIR / "pt_life_expectancy_expected.csv")
+
+@pytest.fixture(scope="session")
+def fr_life_expectancy_expected() -> pd.DataFrame:
+    """Fixture to load the expected output of the cleaning script"""
+    return pd.read_csv(f"{FIXTURES_DIR}/fr_life_expectancy_expected.csv")
+
+
+@pytest.fixture(scope="session")
+def eu_life_expectancy() -> pd.DataFrame:
+    """Fixture to load the expected output of the cleaning script"""
+    return load_data()
```

Comment:
This fixture looks incomplete... Why are you returning your load_data function instead of actual fixture data?